src/content/documentation/develop/manifest-v3-migration-guide.md

```diff
@@ -295,6 +295,7 @@ The format of the top-level manifest.json `version` key in Firefox has evolved a
 - Move any arbitrary strings executed as scripts to files and update your code to use the Scripting API.
 - The add-on ID is required to publish your extension. Make sure to add one in the `browser_specific_settings.gecko.id` manifest.json key.
 - Ensure that the top-level manifest.json `version` key is a string consisting of numbers separated with up to 3 dots. For details, see [version format](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/version/format).
+- Rename the deprecated manifest.json key `applications` to `browser_specific_settings`.
```

Comment:
Put this before line 296. If someone goes through this list in a given order, then it would be nice if they rename applications to bss before adding `bss.gecko.id`. Otherwise they would end up having two declarations of bss.

Reply:
Good idea